spec/models/article_spec.rb

```diff
@@ -25,35 +18,10 @@
   end
 
   describe 'with a published and unpublished article' do
-    after(:each) {
-      Article.destroy_all
-      Page.destroy_all
-    }
     before(:each) {
-      Page.destroy_all
-      Article.destroy_all
-
-      Article.create!({
-                       title: 'Monkey',
-                       body: 'Chicken',
-                       page: 1,
-                       author_name: 'Mike',
-                       published: false
-                      })
-      Article.create!({
-                        title: 'Alligator',
-                        body: 'Chicken',
-                        page: 1,
-                        author_name: 'Jim',
-                        published: true
-                      })
-      Article.create!({
-                        title: 'French Press',
-                        body: 'Chicken',
-                        page: 1,
-                        author_name: 'Megan',
-                        published: true
-                      })
+      create(:article, title: 'Monkey', author_name: 'Mike')
+      create(:article, title: 'Alligator', author_name: 'Jim', published: true)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.